src/main.server.ts

```diff
@@ -3,6 +3,11 @@ import 'zone.js/dist/zone';
 import 'reflect-metadata';
 
 import { enableProdMode } from '@angular/core';
+/******************************************************************
+ * Load `$localize` - used if i18n tags appear in Angular templates.
+ */
+import '@angular/localize/init';
```

Comment:
As we're using ngx-translate and not Angular's own i18n, I don't think this is necessary

Reply:
you right but I had to add it anyway otherwise we get the following error during SSR 
```
ERROR Error: It looks like your application or one of its dependencies is using i18n.
Angular 9 introduced a global `$localize()` function that needs to be loaded.
Please run `ng add @angular/localize` from the Angular CLI.
(For non-CLI projects, add `import '@angular/localize/init';` to your `polyfills.ts` file.
For server-side rendering applications add the import to your `main.server.ts` file.)
```

Reply:
I see. It's a [dependency of ng-bootstrap](https://ng-bootstrap.github.io/#/getting-started#i18n) it seems. 

Perhaps change the comment from `used if i18n tags appear in Angular templates`, to: `not used for i18n in this project, we use ngx-translate. It's used for localization of dates, numbers, currencies, etc.` in order to prevent confusion in the future?